worlds/ladx/LADXR/.gitignore

```diff
@@ -0,0 +1,143 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
I dont think gitignore works in subfolders

Reply:
gitignore does work in subfolders! But this one is probably not needed

Reply:
addressed